### PR TITLE
Added NSwagStudio version 13.13.1.0.

### PR DESCRIPTION
**Pull request by jedieaston**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

------

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/23841)

### CODE REVIEW

**Comment by svrooij** [comment]
This package should not be added, it's broken. **Crash at Startup** see https://github.com/RicoSuter/NSwag/issues/3583

File: manifests/r/RicoSuter/NSwagStudio/13.13.0.0/RicoSuter.NSwagStudio.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: RicoSuter.NSwagStudio
+PackageVersion: 13.13.0.0
```
Comment (svrooij) [inline manifests/r/RicoSuter/NSwagStudio/13.13.0.0/RicoSuter.NSwagStudio.installer.yaml:3]: The mentioned version is broken

**Review by svrooij — CHANGES_REQUESTED** [review]

File: manifests/r/RicoSuter/NSwagStudio/13.13.0.0/RicoSuter.NSwagStudio.installer.yaml (hunk 1 above)
Comment (denelon) [reply manifests/r/RicoSuter/NSwagStudio/13.13.0.0/RicoSuter.NSwagStudio.installer.yaml:3]: You may remove this manifest if this version is broken, but unless the latest one is published, someone else will likely re add this one back.

Comment (jedieaston) [reply manifests/r/RicoSuter/NSwagStudio/13.13.0.0/RicoSuter.NSwagStudio.installer.yaml]: I changed this to use the new 13.13.1.0 version that fixes the crash on startup issue. Thanks for letting me know @svrooij.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**